site/content/tutorial/04-logic/05-keyed-each-blocks/app-a/App.svelte

```diff
@@ -1,23 +1,23 @@
 <script>
 	import Thing from './Thing.svelte';
 
-	let things = [
```

Comment:
I dont see the need of changing the example code over here.

Reply:
I thought it would be more intuitive if the name of the array the `#each` block iterates over is "array". When I read it the first time, I was very confused by so many things being named "things". I mentioned this change in the description.

> Also the array things being named identically to the component <Things> didn't make things easier.